sys/conf/files.arm64

```diff
@@ -163,8 +163,9 @@ arm64/arm64/locore.S		standard	no-obj
 arm64/arm64/machdep.c		standard
 arm64/arm64/machdep_boot.c	standard
 arm64/arm64/mem.c		standard
-arm64/arm64/memcpy.S		standard
-arm64/arm64/memmove.S		standard
+# Morello needs cap-aware memcpy
```

Comment:
These comments should say CHERI as they apply equally to any CHERI-enabled Arm ISA.